src/Tools/CoreFx.Tools/GenerateDepsJson.netstandard.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using System.Text.RegularExpressions;
+using Microsoft.Build.Framework;
+
+namespace Microsoft.DotNet.Build.Tasks
+{ 
+    public partial class GenerateDepsJson: BuildTask
+    {
+        public bool IsNativeAssembly(string file, out AssemblyName result)
```

Comment:
Why can't we just use this version on .NET Framework as well? It looks like we are already using System.Reflection.Metadata, since it's in my Tools/net45 folder.